### ISSUE TITLE
Movepools are severely lacking

### ISSUE THREAD

**Issue by tshadowknight** [tshadowknight/Pokemon-Red-Free#4]

**Comment by greenDarkness** [comment]
I completely agree. It would be nice if Pokemon such as Magikarp and Abra started with a basic attacking move that way you don't have to use Struggle to start with them.
